parsers/DK.py

```diff
@@ -1,250 +1,125 @@
-import json
-import time
 from datetime import datetime, timedelta
 from logging import Logger, getLogger
-from typing import List, Optional
+from typing import List, Optional, Union
 
-import arrow  # the arrow library is used to handle datetimes
-import pandas as pd
-import pytz
-from requests import Session, exceptions
-
-from parsers.lib.config import refetch_frequency
+from requests import Response, Session
 
+from .lib.config import refetch_frequency
 from .lib.exceptions import ParserException
 
-ids = {
-    "real_time": "06380963-b7c6-46b7-aec5-173d15e4648b",
-    "energy_bal": "02356e88-7c4e-4ee9-b896-275d217cc1b9",
+EXCHANGE_MAPPING = {
+    "DE->DK-DK1": {"id": "ExchangeGermany", "direction": 1, "priceArea": "DK1"},
+    "DK-DK1->DK-DK2": {"id": "ExchangeGreatBelt", "direction": -1, "priceArea": "DK1"},
+    "DK-DK1->NL": {"id": "ExchangeNetherlands", "direction": -1, "priceArea": "DK1"},
+    "DK-DK1->NO-NO2": {"id": "ExchangeNorway", "direction": -1, "priceArea": "DK1"},
+    "DK-DK1->SE-SE3": {"id": "ExchangeSweden", "direction": -1, "priceArea": "DK1"},
+    "DK-DK2->SE-SE4": {"id": "ExchangeSweden", "direction": -1, "priceArea": "DK2"},
+    "DK-BHM->SE-SE4": {"id": "BornholmSE4", "direction": -1, "priceArea": "DK2"},
 }
```

Comment:
is DE->DK-DK2 data not available? it is in our exchanges.json and also calls DK.fetch_exchange

Reply:
I must have missed that!
I'll add it and check if it has data right away.

Reply:
Added `DE->DK-DK2`so this should be resolved now. Nice catch.

Reply:
ok thanks Viktor, I will test and approve :)